src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/.template.config/template.json

```diff
@@ -107,6 +173,111 @@
       "datatype": "bool",
       "defaultValue": "false",
       "description": "If specified, includes an ASP.NET Core host for the Blazor app."
+    },
+    "auth": {
+      "type": "parameter",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "None",
+          "description": "No authentication"
+        },
+        {
+          "choice": "Individual",
+          "description": "Individual authentication"
```

Comment:
We were hoping to expose all the same auth types as are available on the server (None, Individual, IndividualB2C, SingleOrg, MultiOrg, Windows). Having done it for the server template, I know it's super tedious putting in the config for all of these.

Do you think that's viable, or do you anticipate any problems with these?

Reply:
Let's take this discussion offline.